src/convenience/webhook.ts

```diff
@@ -252,27 +252,30 @@ function webhookCallback<
 
 function timeoutIfNecessary(
     task: Promise<void>,
-    onTimeout: "throw" | "ignore" | (() => unknown),
+    onTimeout: "throw" | (() => unknown), // need async?
```

Comment:
I think we can change this to be an optional function now, it will either call the function or throw if none is specified